config/settings.yml

```
@@ -18,3 +18,6 @@ alert_sms_account_name: "CDx"
 alert_sms_application_name: "CDx-Dev"
 alert_sms_application_password: "cdx123cdx"
 alert_sms_from_tel: "sms://442393162302"
+recaptcha:
+  site_key: <%= ENV['RECAPTCHA_SITE_KEY'] %>
+  secret_key: <%= ENV['RECAPTCHA_SECRET_KEY'] %>
```

Let's add the ability to disable the feature. It should be opt-in (i.e. disabled by default).

```suggestion
recaptcha:
  enabled: <%= ENV["RECAPTCHA_ENABLED"] == "1" %>
  site_key: <%= ENV["RECAPTCHA_SITE_KEY"] %>
  secret_key: <%= ENV["RECAPTCHA_SECRET_KEY"] %>
```

Of course wherever we use recaptcha we should wrap it with `if Settings.recaptcha.enabled`.